source/community.applications/usr/local/emhttp/plugins/community.applications/skins/Narrow/styles/community.applications.css

```diff
@@ -0,0 +1,2392 @@
+  
+  .ca_bottomLineSpotLight {
+    top: 15rem !important;
+  }
+  
+  .ca_categories {
+    font-size: 1rem;
+    font-style: italic;
+  }
+  
+  .cardCategory {
+    font-size: 1rem;
+    margin-left: 0.75rem;
+  }
+  
+  .cardDescription {
+    /* Layout */
+    display: block;
+    position: relative;
+    top: 0.5rem;
+    max-height: 7rem;
+    
+    /* Interactive */
+    cursor: pointer;
+    overflow: hidden;
+  }
+  
+  .cardDesc {
+    display: inline-block;
+    max-height: 6rem;
+    overflow: hidden;
+  }
+  
+  .cardWarning {
+    font-size: 1.5rem;
+    color: #bdbd14;
+  }
+  
+  /* Popup Button Styles */
+  .caButton {
+    /* Layout */
+    display: inline-block;
+    margin-right: 1rem;
+    padding: 1px 10px;
+    
+    /* Typography */
+    font-size: 1.5rem;
+    line-height: 2rem;
+    color: var(--support-popup-text) !important;
+    
+    /* Visual */
+    background: linear-gradient(
+      to bottom,
+      transparent 0%,
+      rgba(0, 0, 0, 0.4) 100%
+    ),
+    linear-gradient(
+      to right,
+      color-mix(in srgb, var(--support-popup-background) 85%, white) 0%,
+      var(--support-popup-background) 50%,
+      color-mix(in srgb, var(--support-popup-background) 85%, white) 100%
+    );
+    background-position: 0 0;
+    background-size: 100% 100%;
+    border-radius: 15px;
+    
+    /* Interactive */
+    cursor: pointer;
+  }
+  
+  .caButton a {
+    text-decoration: none;
+    color: var(--support-popup-text);
+    cursor: pointer;
+  }
+  
+  .caButton:hover {
+    background: var(--unraid-66-color);
+    color: #fff !important;
+  }
+  
+  .caButton:hover a,
+  a.caButton:hover {
+    color: #fff;
+    text-decoration: none;
+  }
+  
+  .maxPerPage {
+    position: absolute;
+    right: 0;
+  }
+  
+  #dropdown-maxPerPage {
+    position: absolute;
+    right: 0;
+  }
+
+  
+  /* Form Elements */
+  input[type="checkbox"] {
+    width: 2rem;
+    height: 2rem;
+    margin-left: 0;
+  }
+  
+  input[type="button"] {
+    background: none;
+    font-size: 1.5rem;
+  }
+  
+  input:hover[type="button"] {
+    color: var(--unraid-66-color) !important;
+    background: none !important;
+  }
+  
+  input:hover[type="button"][disabled] {
+    background: none !important;
+    color: currentColor !important;
+  }
+  
+  input[type="button"][disabled] {
+    background: none !important;
+  }
+  
+  .searchSubmit {
+    /* Layout */
+    width: 2rem;
+    min-width: 2rem;
+    height: 3.4rem;
+    padding: 1.1rem 1rem 1rem 0;
+    position: relative;
+    
+    /* Typography */
+    font-family: 'FontAwesome';
+    font-size: 1.1rem;
+    
+    /* Visual */
+    background: var(--template-hover-background);
+    border: none;
+    
+    /* Interactive */
+    cursor: pointer;
+  }
+  
+  #searchButton:hover {
+    color: var(--unraid-66-color);
+  }
+  
+  /* Menu and Navigation */
+  .menuHeader {
+    font-size: 2rem;
+    margin: 1rem 0;
+  }
+  
+  .menuItems {
+    position: relative;
+    top: 2rem;
```

Comment:
Using `top`, `right`, `bottom`, or `left` with `position: relative` is going to give you some headaches and throw off the flow of things where you will have to unexpectedly over compensate for other in elements.

Try using `margin-top` instead.